### PR TITLE
Tdr 491 ecr repositories

### PR DESCRIPTION
**Pull request by MancunianSam**

### CODE REVIEW

File: ecr/templates/auth_server_policy.json.tpl [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+  "Version": "2008-10-17",
+  "Statement": [
+    {
+      "Sid": "AllowPullConsignmentAPI",
+      "Effect": "Allow",
+      "Principal": {
+        "AWS": [
+          "arn:aws:iam::${intg_account}:role/keycloak_ecs_execution_role_intg",
```
Comment (TomJKing) [inline ecr/templates/auth_server_policy.json.tpl:9]: Do we need the production account here aswell?

**Review by TomJKing — COMMENTED** [review]
One very minor comment/question

File: ecr/templates/auth_server_policy.json.tpl (hunk 1 above)
Comment (MancunianSam) [reply ecr/templates/auth_server_policy.json.tpl:9]: We can't. If the role doesn't exist you get an error. For most things you can get away with non-existent role it seems but in this case, AWS checks.

Comment (TomJKing) [reply ecr/templates/auth_server_policy.json.tpl:9]: Fair enough.

**Review by TomJKing — APPROVED** [review]

**Pull request merged**